### ISSUE TITLE
Error on /trait sentry

### ISSUE THREAD

**Issue by Culvanen** [jrbudda/Sentry#245]
Hi,

I use a server 1.8.3 Spigot

I also Sentry Citizen and Citizen working properly but the assignment of Sentry related to an NPC causes an error:

"Java.lang.NoClassDefFoundError: net / minecraft / server / v1_8_R2 / Packet"

I think I have the right version of the plugins, if you think the problem can come in versions thank you to tell me the correct versions because it's so hard to follow (...)

Thank you for your work and for your help

cordially

**Comment by mcmonkey4eva** [comment]
You're not running Spigot 1.8.3 if you get that error. Update your server.

**Comment by Culvanen** [comment]
Work fine, thx !
